Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,965 @@
+        up extensions.
+
+    ..  _allow-fields:
+    -   *Allow Fields*
+
+        Allow fields refer to fields of content elements displayed in the TYPO3 backend in regard to
+        their permissions. Editors can only edit fields in the backend, which are included in the
+        list of "Allow fields" in their permission setup.
+
+    ..  _assets:
+    -   *Assets*
+
+        Assets are media resources like images, videos and documents that are
+        uploaded and managed in the TYPO3 system. Also, extensions can include assets which
+        can be referred to in the frontend, like specific icons or even JavaScript libraries.
+
+*   B
+
+    ..  _backend---frontend:
+    -   *Backend / Frontend*
+
+        Backend and Frontend are the two main areas of TYPO3 CMS. The backend is
+        the administrative interface for editors and administrators, while the
+        frontend is the publicly accessible part of the website.
+
+    ..  _backend-bookmarks:
+    -   *Backend Bookmarks*
+
+        Backend bookmarks are shortcuts that users can set for frequently used
+        backend pages for quicker access.
+
+    ..  _backend-layout:
+    -   *Backend Layout*
+
+        The backend layout defines the structure and design of the backend user
+        interface in regard to maintaining content elements and the arrangement of their
+        input fields. A backend layout can be set on a page-level, and this attribute
+        can actually be also evaluated in the frontend, to affect the arrangement of
+        elements on a page.
+
+    ..  _backend-module:
+    -   *Backend Module*
+
+        Backend modules are extendable components in the TYPO3 backend that
+        provide various functionalities and tools like user management or file
+        management. The left hand panel in the backend shows all these modules.
+
+*   C
+
+    ..  _callout:
+    -   *Callout*
+
+        A callout is a highlighted element designed to draw attention to important
+        information or actions.
+
+    ..  _cache--cache-backend--frontend-cache:
+    -   *Cache (Cache Backend, Frontend Cache)*
+
+        Caches are used to improve website performance by storing frequently
+        accessed data. The backend cache stores data in the administration system,
+        while the frontend cache stores data for the website display.
+
+    ..  _cache-tags:
+    -   *Cache Tags*
+
+        Cache Tags are labels assigned to cache entries to facilitate the
+        management and invalidation of cached data.
+
+    ..  _certification--t-cc--t-cd--t-ci--t-ce:
+    -   *Certification (T3CC, T3CD, T3CI, T3CE)*
+
+        Certifications in the TYPO3 ecosystem, such as T3CC (Consultant), T3CD (Developer),
+        T3CI (Integrator), and T3CE (Editor) confirm the proficiency of developers and
+        integrators in various aspects of TYPO3 CMS. TYPO3 has an official certification
+        strategy.
```

Comment:
It's not `T3` for `TYPO3`, it's `TC` for `TYPO3 CMS`:
https://typo3.com/services/certifications

```suggestion
    -   *Certification (TCCC, TCCD, TCCI, TCCE)*

        Certifications in the TYPO3 ecosystem, such as TCCC (Consultant), TCCD (Developer),
        TCCI (Integrator), and TCCE (Editor) confirm the proficiency of developers and
        integrators in various aspects of TYPO3 CMS. TYPO3 has an official certification
        strategy.
```